esp/bindings/http/platform/httptransport.cpp

```diff
@@ -798,6 +798,44 @@ void CHttpMessage::logSOAPMessage(const char* message, const char* prefix)
     return;
 }
 
+static bool skipLogContent(const char* httpHeader)
+{
+    VStringBuffer postStr("%s ", POST_METHOD);
+    if (!startsWith(httpHeader, postStr.str()))
+        return false;
+
+    const char* servicePtr = httpHeader + 5;
+    if (isEmptyString(servicePtr) || (servicePtr[0] != '/'))
+        return false;
+
+    const char* methodPtr = strchr(++servicePtr, '/');
+    if (!methodPtr)
+        return false;
+
```

Comment:
I think you could speed up almost all calls to this function by checking here if servicePtr begins with "ws_access/" or "ws_account/" or not.  If not, I don't think there is any reason to continue?  That will usually be a fast exit.